flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -78,6 +78,12 @@ under the License.
 		</dependency>
 		-->
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
```

Comment:
This should be incorrect; as is it would be bundled in the user-jar.

This also shows the annoying side of this change. If a user wants to run things in the IDE/tests they now have to add another dependency to their poms (which they have no way of figuring out).